### PR TITLE
feature: command for tca key binding problems

### LINKED ISSUE #431: TCA command to better document the preferred settings of the TCA Throttle quadrant and addons
### Feature type

Support information

### Description

The control settings in MSFS for the TCA throttle quadrant requires removing certain settings and to properly set up the TCA addons so that it doesn't interfere. 

Focus is mostly on flaps and spoilers that would get triggered in case of pause/unpause if they are bound, but the addons aren't present. 

### References (optional)

https://docs.flybywiresim.com/fbw-a32nx/a32nx-api/hardware/?h=tca#thrustmaster-tca-throttle-add-on-tca-quadrant-add-on-airbus-edition
https://ts.thrustmaster.com/download/pictures/Mappings/Mapping-TCAQuadrant_And_AddOn_MFS.png

### Additional info (optional)

_No response_

### Discord Username (optional)

straks#7240

### PR DESCRIPTION
**Pull request by Sleinmaster**
## Description

Adds a new command .tca to give a hint to remove all unused key bindings if the tca quadrant is used without the add-ons

fixes #431 
## Test Results

Tested without the image. Image and text certainly need some improvement:
image

## Discord Username

Slein#8982

### CODE REVIEW

**Comment by pdellaert** [comment]
Only comment I have would be : Do we need each sentence on a single line, or can we just make it a single paragraph/line? (same text still). 

Screenshot: Will need to find an example of the default configuration. Might check if I can get that happen somehow.

File: src/commands/support/tca.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { MessageCommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+
+const TCA_URL = `${process.env.IMAGE_BASE_URL}support/tca-settings.png`;
```
Comment (pdellaert) [inline src/commands/support/tca.ts:5]: filename of the image is different from the actual name.

**Review by pdellaert — COMMENTED** [review]
We might want to change the image to higher quality and perhaps without the throttle itself? Maybe I'll look into it this weekend and also look at the doc side. Then we can merge it together.

**Comment by benw202** [comment]
@Sleinmaster - Test results state image and text need improvement, is this still the case or is this PR ready to go?

File: src/commands/support/tca.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+const tcaEmbed = makeEmbed({
+    title: 'FlyByWire Support | TCA Throttle',
+    description: makeLines([
+        'The Thrustmaster TCA Quadrant Airbus Edition is a popular hardware throttle for flying the FlyByWire A32NX. The Quadrant can be bought and used with or without the add-ons which include axis and buttons for flaps, spoilers, gear, and more.',
+        'The default controller settings in MSFS always includes the key bindings for the add-on axis and buttons aswell which can cause problems if the add-ons are not connected, e.g., automatically deploying full flaps, especially after pausing the sim.',
```
Comment (pdellaert) [inline src/commands/support/tca.ts:11]:
```suggestion
        'The default controller settings in MSFS always includes the key bindings for the add-on axis and buttons, which can cause problems if the add-ons are not connected, e.g., automatically deploying full flaps, especially after pausing the sim.',
```
Minor suggestion

File: assets/images/support/tca-settings.png
Comment (pdellaert) [inline assets/images/support/tca-settings.png:1]: Propose to use this slightly scaled down and optimized version
tca-settings_optimized

**Review by pdellaert — CHANGES_REQUESTED** [review]
Small change and image update, otherwise ready to go!

**Comment by Sleinmaster** [comment]
@benw202 
New screenshot
image

@pdellaert thanks for the review and the new image, just wanted to ask you.

**Review by pdellaert — APPROVED** [review]
I'll let Ben do the final check, but looking good here! (Does need a date for the changelog, but that can be done right before merging)

**Pull request merged**